plugins/inputs/intel_pmt/sample.conf

```diff
@@ -0,0 +1,16 @@
+# Intel Platform Monitoring Technology plugin exposes Intel PMT metrics available through the Intel PMT kernel space.
+# This plugin ONLY supports Linux.
+[[inputs.intel_pmt]]
+  ## URL or filepath to PMT XML within PMT repository.
+  ## Both URL and filepath should be absolute.
+  pmt_source = "https://raw.githubusercontent.com/intel/Intel-PMT/master/xml/pmt.xml"
+  
+  ## Filter metrics by their datatype.
+  ## See the Filtering section in README for more details.
+  ## If empty, all metrics are collected.
+  # datatype_metrics = []
```

Comment:
Wouldn't naming this `datatype_filter` make more sense? Same for the `sample_metrics`...

Reply:
renamed to `datatypes_enabled` and `samples_enabled` actually